viewing_party/party.py

```diff
@@ -1,23 +1,126 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    # Only create/return a movie dict if all arguments are valid (truthy)
+    # Assumes rating is greater than 0
+    if all([title, genre, rating]):
+        return {
+            "title": title,
+            "genre": genre,
+            "rating": rating,
+        }
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    watched_list = user_data["watched"]
+    watchlist = user_data["watchlist"]
+
+    # Iterate through each movie in watchlist, looking for a match to the argument title
+    for index, movie in enumerate(watchlist):

```

Comment:
Nice use of `enumerate`! You could also have done a `for movie in watchlist` syntax, but I see you used the `index` here so this totally works.